### PR TITLE
fix: ga4- fix total to price for events

### PR DESCRIPTION
**Pull request by yashasvibajpai**
## PR Description

We are fixing sending price for supported events instead of total.
- product viewed
- product added
- product removed
- add to wishlist

## Notion ticket

Ticket link

## Screenshots

Please add screenshots for any new features or UI bug fixes for the following browsers -

- Chrome
  >
- Firefox
  >
- Safari
  >

## Security

- [ ] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

**Review by shrouti1507 — DISMISSED** [review]

File: src/integrations/GA4/ECommerceEventConfig.js [hunk 1]
```diff
@@ -169,7 +169,7 @@ const eventNamesConfigArray = [
     hasItem: true,
     includeList: [
       eventParametersConfigArray.Currency,
-      eventParametersConfigArray.Total,
+      eventParametersConfigArray.Price,
```
Comment (ItsSudip) [inline src/integrations/GA4/ECommerceEventConfig.js:172]: It looks like a breaking change. Can't we support both the total and price?
```suggestion
      eventParametersConfigArray.Total,
      eventParametersConfigArray.Price,
```

Comment (yashasvibajpai) [reply src/integrations/GA4/ECommerceEventConfig.js:172]: We could, but that essentially means we are deviating from the rudderstack ecommerce spec.
For example, in this event (Product Viewed), we do not support a total property in the root ([reference](https://www.rudderstack.com/docs/event-spec/ecommerce-events-spec/ordering/#product-viewed)) and rather support a `price` property. 
We could though keep it for backward compatibility purposes, if this is okay.

Comment (ItsSudip) [reply src/integrations/GA4/ECommerceEventConfig.js:172]: Let's keep it backword compatible.

Comment (yashasvibajpai) [reply src/integrations/GA4/ECommerceEventConfig.js]: Done

**Review by ItsSudip — APPROVED** [review]

**Review by shrouti1507 — APPROVED** [review]

**Review by sandeepdsvs — APPROVED** [review]

**Pull request merged**